server/lib/validation/languages/ko.js

```diff
@@ -0,0 +1,54 @@
+// Minimum of characters that qualify as a sentence.
+const MIN_CHARACTERS = 1;
+
+// Maximum of characters allowed per sentence to keep recordings in a manageable duration.
+const MAX_CHARACTERS = 50;
+
+const INVALIDATIONS = [{
+  fn: (sentence) => {
+    // To properly tokenize Korean, We need some heavy tokenizers (ex: mecab-ko, nori, ...),
+    // For counting letters those tokenizers are not necessary.
+
+    // Before counting letters, normalization should be applied, since length may be tripled
+    const normalizedSenetence = sentence.normalize('NFC');
+    return normalizedSenetence.length < MIN_CHARACTERS || normalizedSenetence.length > MAX_CHARACTERS;
+  },
+  error: `Number of characters must be between ${MIN_CHARACTERS} and ${MAX_CHARACTERS} (inclusive)`,
```

Comment:
For some other validators we started to show the errors in the respective languages. Might be worth to do this here as well :)